demisto_sdk/commands/common/tools.py

```diff
@@ -144,7 +144,7 @@ def run_command(command, is_silenced=True, exit_on_error=True, cwd=None):
     return output
 
 
-def get_remote_file(full_file_path, tag='master'):
+def get_remote_file(full_file_path, tag='master', return_content=False):
```

Comment:
`return_content` returns the raw content of the file and not the dict representation of it?
let's add docstring to this func.